components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1365,8 +1372,13 @@ struct BitcoinAccountInfo {
   uint64 balance;
 };
 
+struct BitcoinBalance {
+  uint64 total_balance;
+  map<string, uint64> balances;
+};
+
 interface BitcoinWalletService {
-  GetBalance(string network_id, AccountId account_id) => (string balance, string error_message);
+  GetBalance(string network_id, AccountId account_id) => (BitcoinBalance? balance, string? error_message);
```

Comment:
Let's add some comments for `GetBalance`. For example, are unconfirmed balances included?